homeassistant/components/vallox/__init__.py

```diff
@@ -110,9 +110,6 @@
                                      service_handler.async_handle,
                                      schema=schema)
 
-    # Fetch initial state once before bringing up the platforms.
```

Comment:
Maybe add a comment above scheduling the state update, why we don't fetch state during setup.